working-group-abilities.md

```diff
@@ -0,0 +1,36 @@
+# Working Group Abilities
+
+Established working groups in the OSSF are given broad latitude to accomplish their goals.
+The TAC aims to only require formal approval in limited cases.
+This document outlines some of the explicitly permitted activities and resources made
+available to working groups.
+It is not intended to be exhaustive, and will grow over time.
+
+When in doubt, reach out to the TAC with any questions!
+
+## Repositories
+
+We expect that working groups will need to create repositories for code, tools, and other projects.
+These should be created in the [ossf](github.com/ossf) organization where possible to simplify
+management, but working groups can also create new organizations where necessary.
```

Comment:
Logistically, I think the working groups will need to request repos from an ossf github org admin (someone in LF).
I don't think github has a mult-tier organization feature that would allow a more autonomous process, i.e. working groups have their own organizations under ossf governance. Similarly there might not be useful mechanics yet to enable working groups to create additional github organizations that are connected to ossf.
So long winded way of saying we should strike "working groups can also create new organizations where necessary" :)

Reply:
> Logistically, I think the working groups will need to request repos from an ossf github org admin

Right now this is actually pretty much everyone in the org - all members can create public repos:
![image](https://user-images.githubusercontent.com/1714486/104763117-0be99600-572b-11eb-8d25-d2f65b0320b8.png)

I'd be in favor of restricting this to WG leads if we want to later on.

> Similarly there might not be useful mechanics yet to enable working groups to create additional github organizations that are connected to ossf.

For orgs, the CVE benchmark project setup another org already over here: https://github.com/ossf-cve-benchmark. The flow actually went in reverse - they created the org and then added Lindsay to it to transfer to LF.

Maybe we can change that sentence to:

> Working groups can also request new organizations where necessary - reach out to the TAC for logistics.